index.html

```diff
@@ -1283,6 +1283,9 @@ <h3>Contexts</h3>
 the Verifiable Credentials Data Model is available at
 <a href="https://www.w3.org/2018/credentials/">https://www.w3.org/2018/credentials/</a>.
 This concept is further expanded on in Section <a href="#extensibility"></a>.
+Specification authors MAY require that JSON-LD Contexts are integrity protected
+by using the `relatedResource` feature described in Section
+[[[#integrity-of-related-resources]]], or an equivalent mechanism.
```

Comment:
Acting on @iherman's comment
```suggestion
Authors of JSON-LD Contexts MAY require that those context files be integrity 
protected by using the `relatedResource` feature described in Section
[[[#integrity-of-related-resources]]] or an equivalent mechanism.
```